include/deal.II/grid/grid_tools.h

```diff
@@ -680,6 +752,21 @@ namespace GridTools
   compute_local_to_global_vertex_index_map(
     const parallel::distributed::Triangulation<dim,spacedim> &triangulation);
 
+  /**
+   * Return the highest value among ratios between dimensions of a
```

Comment:
to avoid confusion, maybe replace "dimension" here by "extents in each of the coordinate directions"?